### DISCUSSION TITLE
Is anything like Rails console possible with Prisma?

### DISCUSSION THREAD

**Discussion by axhamre** [prisma/prisma#6300] [Q&A] [answered]
Is it possible to execute Prisma queries in some kind of REPL environment - for example how Rails console works?

**Comment by ryands17** [comment, marked as answer]
@axhamre 👋 

Prisma directly doesn't support this, but you might want to check out Blitz that has this: https://blitzjs.com/docs/tutorial#playing-with-the-prisma-database-client

**Reply by axhamre** [reply]
Amazaing, thank you! 🙏

**Comment by joshwilsonvu** [comment]
Would be core team be interested in a PR for a `prisma console` similar to Blitz's? I've found `rails console` very useful and thought it would be nice to have a way to quickly interact with the DB, or run a quick script. 

I'm also thinking you could use [custom commands](https://nodejs.org/api/repl.html#repl_replserver_definecommand_keyword_cmd) like `.db`, `.generate`, etc. that delegate to the Prisma CLI, so that users can interleave running queries and CLI commands without context switching.

If you are interested and have any other requests, let me know and I can get started. Thanks!

**Reply by janpio** [reply]
I think we would be interested in a issue as a feature request, that describes the user case and so on a bit more. Then we could probably refine the possible solutions a bit as well before you jump into implementation. 

In general I personally think a REPL for Prisma queries could be a super interesting thing indeed.

**Reply by axhamre** [reply]
Created issue: https://github.com/prisma/prisma/issues/9563

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
